api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,72 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
```

Comment:
let's document default behaviour and let the backend default to true.
A bool here will need an lint exception, see https://github.com/openshift/hypershift/blob/main/api/.golangci.yml#L118-L121
for an example